auth0/resource_auth0_organization.go

```diff
@@ -64,7 +60,6 @@ func newOrganization() *schema.Resource {
 			"connections": {
 				Type:     schema.TypeSet,
 				Optional: true,
-				Computed: true,
```

Comment:
This doesn't need to be a computed property, it's actually us within the config that decide the values of these and then it gets created on the API.